components/Header.vue

```diff
@@ -45,7 +45,9 @@
 <script>
 // TODO
 // Want to make this more of a modal
-import Aside from "~/components/Aside"
+import Brand from "~/components/Brand"
+import NavBar from "~/components/NavBar"
+import SearchForm from "~/components/SearchForm"
 export default {
 	components: {
 		Aside
```

Comment:
@TeddTech I notice that you aren't importing Aside any more, but there is a tag and this reference to it still. Also not sure if this line is valid. If these are errors, could you please file a new Issue to clean them up.

Reply:
@jobara This is resolved in https://github.com/inclusive-design/wecount.inclusivedesign.ca/pull/52/files